dts/arm/nordic/nrf51822.dtsi

```diff
@@ -91,7 +91,7 @@
 		gpio0: gpio@50000000 {
 			compatible = "nordic,nrf-gpio";
 			gpio-controller;
-			reg = <0x50000000 0x800>;
+			reg = <0x50000000 0x1000>;
```

Comment:
I'm wondering if we shouldn't use the same for nRF52810 and nRF52832, and leave these sophisticated split `reg` definitions only for `gpio0` and `gpio1` in nRF52840, where they are actually needed (ideally with a comment why such complicated definitions are used).

Reply:
this is good point. I will update it.

Reply:
done